### PR TITLE
remove "add new garden" on other member's page

### PR DESCRIPTION
**Pull request by Thomascountz**
#1167 

`if @garden.owner == current_member` doesn't keep in kind with using cancan's ability methods. 

Updating to `can? :edit @garden` would also solve the problem, but maybe doesn't express intent clearly enough?

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -76,7 +76,5 @@ submit the change with your pull request.
 - Brenda Wallace / [br3nda](https://github.com/br3nda)
 - Jim Stallings / [jestallin](https://github.com/jestallin)
 - Alyssa Ransbury / [alran](https://github.com/alran)
-
-## Bots
-
+- Thomas Countz / [thomascountz](https://github.com/thomascountz)
```
Comment (Br3nda) [inline CONTRIBUTORS.md]: You've accidentally removed the # Bots header here.

Comment (Thomascountz) [inline CONTRIBUTORS.md]: Oops! I think I was working with an old version of that file. Sorry! 

Is that something I should fix?

Comment (Br3nda) [inline CONTRIBUTORS.md]: yeah, if you `git rebase` you'll get a git conflict which you can resolve and re-instate those lines.

Comment (Br3nda) [inline CONTRIBUTORS.md]: or.. you prefer, just edit the file, add `## Bots` back in, and commit and push

**Comment by Br3nda** [comment]
Note: fixes #1167

**Comment by Thomascountz** [comment]
Awesome, thanks @Br3nda, I pushed a new commit.

**Comment by Thomascountz** [comment]
Not sure if the checks are failing because of something I did...
This is my first PR, sorry for the hassle!

**Pull request merged**